share/management/commands/removeanonymousworks.py

```diff
@@ -0,0 +1,26 @@
+from django.core.management.base import BaseCommand
+from django.db.models import Exists
+from django.db.models import OuterRef
+
+from share.search import SearchIndexer
+
+from project import celery_app
+
+from share.models import WorkIdentifier
+from share.models import AbstractCreativeWork
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        qs = AbstractCreativeWork.objects.annotate(
```

Comment:
Is annotate + exclude better than just `.filter(identifiers=None)` or `.filter(identifiers__isnull=True)`?